internal/core/application/account_service.go

```diff
@@ -70,90 +74,90 @@ func (as *AccountService) CreateAccountBIP44(
 }
 
 func (as *AccountService) DeriveAddressesForAccount(
-	ctx context.Context, accountName string, numOfAddresses uint64,
+	ctx context.Context, namespace string, numOfAddresses uint64,
```

Comment:
According to the changes above, this should become a generic `name`.
```suggestion
	ctx context.Context, name string, numOfAddresses uint64,
```